### DISCUSSION TITLE
enable_ssl.sh failing

### DISCUSSION THREAD

**Discussion by BuzzerBoy** [ant-media/Ant-Media-Server#6720] [General]
I'm trying to install ant media server on Ubuntu 22.04.  
Whenever I try to run the enable_ssl.sh script, I get the following error:

**./enable_ssl.sh: line 208: check_marketplace: command not found
There is a problem with the script. Please re-run the enable_ssl.sh script.**

Here is the full command line text:

**fremantlemedia@ant-server:~$ cd /usr/local/antmedia/ && sudo ./enable_ssl.sh**
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
jq is already the newest version (1.6-2.1ubuntu3).
iptables is already the newest version (1.8.7-1ubuntu5.2).
iptables set to manually installed.
The following NEW packages will be installed:
  dnsutils
0 upgraded, 1 newly installed, 0 to remove and 1 not upgraded.
Need to get 3,924 B of archives.
After this operation, 60.4 kB of additional disk space will be used.
Get:1 http://us.archive.ubuntu.com/ubuntu jammy-updates/universe amd64 dnsutils all 1:9.18.28-0ubuntu0.22.04.1 [3,924 B]
Fetched 3,924 B in 1s (6,418 B/s)    
Selecting previously unselected package dnsutils.
(Reading database ... 181228 files and directories currently installed.)
Preparing to unpack .../dnsutils_1%3a9.18.28-0ubuntu0.22.04.1_all.deb ...
Unpacking dnsutils (1:9.18.28-0ubuntu0.22.04.1) ...
Setting up dnsutils (1:9.18.28-0ubuntu0.22.04.1) ...
./enable_ssl.sh: line 208: check_marketplace: command not found
There is a problem with the script. Please re-run the enable_ssl.sh script.

Someone else has posted this error, but there was no response.

**Comment by muratugureminoglu** [comment]
Hi @BuzzerBoy 

Could you please try using the following script ? You can copy the file under /usr/local/antmedia/

https://raw.githubusercontent.com/ant-media/Ant-Media-Server/refs/heads/master/src/main/server/enable_ssl.sh

Please don't forget to give some permission as follows.

```
chmod +x /usr/local/antmedia/enable_ssl.sh
```

Please let me know If It does not work for you.

Regards.

**Comment by BuzzerBoy** [comment]
This script worked perfectly.  Thank you.  You may want to add to your
quickstart documentation that you have to have port 80 open for inbound
traffic to get the ssl cert.

I have the server up and running for testing now. Another question,
I have an SDI capture card (Magewell) in my machine.  When I use OBS to
push an RTMP stream into Ant Media server it looks fine.  When I use our
"liveapp" sample webpage for ingest into the server, it squishes the
widescreen video format into a square format and distorts the image on
playback.  Is there a tweak I can make to the sample ingest webpage that
allows for widescreen (1080p) video?

Thanks!
John

**Reply by muratugureminoglu** [reply]
Hi @BuzzerBoy 

Thanks for your feedback. 

You can find a list of ports that need to be opened in our documentation[1]

[1] https://antmedia.io/docs/guides/installing-on-linux/installing-ams-on-linux/#server-ports

For your other question, please create a separate thread, as your question can be easily found and may help others.

Thanks.
